hash/megadriv.xml

```diff
@@ -26638,6 +26649,11 @@ Black screen, writes in ROM area then jumps to PC=1a2000 (encryption?)
 	</software>
 
 	<software name="sonic2p2" cloneof="sonic2">
+		<!--
+			Presumably built sometime between June and early August of 1992.
+			Originally found in a pirate cartridge, an official prototype with this build has been found and proved to match.
+			TODO: Dump needs to be further documented and split accordingly.
+		-->
```

Comment:
`notes`, also the `TODO` is implicit from the ROM loading context, get rid of it.

Reply:
Done.